tests/testthat/test_srp_model_impute.R

```diff
@@ -123,3 +123,21 @@ test_that("Testing if a single indivudual can be sampled only once in a group",
   # there should only be a single individual, single iteration
   expect_true(nrow(tbl) == 1)
 })
+
+test_that("Testing speed", { # nolint
```

Comment:
delete the test case, no need to test speed

Reply:
done